code/modules/mob/living/carbon/human/death.dm

```diff
@@ -79,6 +79,18 @@
 	if(is_hivehost(src))
 		var/datum/antagonist/hivemind/hive = mind.has_antag_datum(/datum/antagonist/hivemind)
 		hive.destroy_hive()
+	if(HAS_TRAIT(src, TRAIT_DROPS_ITEMS_ON_DEATH))
+		var/list/contents = get_contents()
```

Comment:
Also contents should already be defined by byond /atom, so removing this line is probably fine